src/Universum/Unsafe.hs

```diff
@@ -0,0 +1,24 @@
+{-# LANGUAGE Unsafe #-}
+
+-- | Unsafe functions to work with lists and 'Maybe'.
+-- Sometimes unavoidable but better don't use them. This module
+-- is not even included in default prelude exports.
+
+module Universum.Unsafe
+       ( head
+       , tail
+       , init
+       , last
+       , at
+       , (!!)
+       , fromJust
+       ) where
+
+import Data.List (head, init, last, tail, (!!))
+import Data.Maybe (fromJust)
+
+import Universum.Base (Int)
+
+-- | Alias for '!!'.
+at :: [a] -> Int -> a
+at = (!!)
```

Comment:
I think about it a little bit more. And it seems better to me to flip arguments of `at` function. To be more consistent with `splitAt` `take` `drop` and other functions.